osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -390,6 +390,9 @@ protected override void PopIn()
 
         protected override bool OnKeyDown(KeyDownEvent e)
         {
+            // don't absorb control as ToolbarRulesetSelector uses control + number to navigate
+            if (e.ControlPressed) return false;
```

Comment:
I don't really like this solution, but given that the "proper" way to do this (`KeyBindingContainer` - not even necessarily databased) is a lot more long-winded I guess I could stomach it if everyone else can.